### PR TITLE
changed and fixed

### PR DESCRIPTION
**Pull request by jamshidbek7375**

### CODE REVIEW

File: ExpenseTracker.Api/ExpenseTracker.Application/Services/WalletService.cs [hunk 1]
```diff
@@ -72,6 +72,8 @@ public async Task UpdateAsync(UpdateWalletRequest request)
         ArgumentNullException.ThrowIfNull(request);
 
         var wallet = _mapper.Map<Wallet>(request);
+        wallet.OwnerId = _currentUserService.GetUserId();
+        wallet.CreatedBy = _currentUserService.GetUserName();
```
Comment (Ramadan1011) [inline ExpenseTracker.Api/ExpenseTracker.Application/Services/WalletService.cs:76]: I think, wallet owner does not change. And you wrote unnecessary code.
_Createdby_ and _Updatedby_ are always given values ​​when running ### SaveChangesAsync in **AuditInterceptor**

File: ExpenseTracker.Api/ExpenseTracker.Domain/Common/AuditableEntity.cs [hunk 2]
```diff
@@ -3,7 +3,7 @@
 public abstract class AuditableEntity : EntityBase
 {
     public DateTime CreatedAtUtc { get; set; }
-    public required string CreatedBy { get; set; }
+    public string? CreatedBy { get; set; }
```
Comment (Ramadan1011) [inline ExpenseTracker.Api/ExpenseTracker.Domain/Common/AuditableEntity.cs:6]: createdby shouldn't be nullable

**Review by Ramadan1011 — COMMENTED** [review]
dicuss my review with others

File: ExpenseTracker.Api/ExpenseTracker.Domain/Common/AuditableEntity.cs (hunk 2 above)
Comment (jamshidbek7375) [reply ExpenseTracker.Api/ExpenseTracker.Domain/Common/AuditableEntity.cs]: I corrected.

**Review by Ramadan1011 — APPROVED** [review]
good job!

**Review by davron41 — APPROVED** [review]
looks good

**Review by Abdulazim1771 — APPROVED** [review]

**Pull request merged**